node/include/cocaine/detail/service/node/engine.hpp

```diff
@@ -17,13 +17,16 @@
 #include "cocaine/detail/service/node/slave/load.hpp"
 #include "cocaine/detail/service/node/stats.hpp"
 
+#include "node/pool_observer.hpp"
```

Comment:
C'mon, placing headers in detail has a lot of disadvantages. And here is the right place to start something better without meaningless large-commit-that-changes-everything. Besides, there is no code style strictly speaking.